conv_model_bwd.cpp

```diff
@@ -336,6 +339,22 @@ int conv_benchmark(int argc, char** argv) {
               }
             }
             brgemm_kernel.gemm( &gemm_param );
+
+            /* Zero Rim..  */
+            if (avoid_rim_pollution && h_in_gemm > 1 && i_r == R-r_step && i_s == S-s_step && i_h == ofh-h_step && i_w == ofw-w_step && i_k == Kb - Kb_step) {
```

Comment:
The added zeroing for the rims, done at the end of processing of [n, cb] block of grad input.